### ISSUE TITLE
cannot build wheel

### ISSUE THREAD

**Issue by ScDor** [optapy/optapy#15]
Hi,

When following the installation instructions and calling `python -m build`, I get the following error:

```
...
* Building wheel...
running bdist_wheel
running build
running build_py
Traceback (most recent call last):
  File "C:\Users\User\ProjectPath\venv\lib\site-packages\pep517\in_process\_in_process.py",
line 349, in <module>
    main()
  File "C:\Users\User\ProjectPath\venv\lib\site-packages\pep517\in_process\_in_process.py",
line 331, in main
    json_out['return_val'] = hook(**hook_input['kwargs'])
  File "C:\Users\User\ProjectPath\venv\lib\site-packages\pep517\in_process\_in_process.py",
line 248, in build_wheel
    return _build_backend().build_wheel(wheel_directory, config_settings,
  File "C:\Users\User\AppData\Local\Temp\build-env-lkyfb7qt\lib\site-packages\setuptools\build_meta.py", line 221
, in build_wheel
    return self._build_with_temp_dir(['bdist_wheel'], '.whl',
  File "C:\Users\User\AppData\Local\Temp\build-env-lkyfb7qt\lib\site-packages\setuptools\build_meta.py", line 207
, in _build_with_temp_dir
    self.run_setup()
  File "C:\Users\User\AppData\Local\Temp\build-env-lkyfb7qt\lib\site-packages\setuptools\build_meta.py", line 150
, in run_setup
    exec(compile(code, __file__, 'exec'), locals())
  File "setup.py", line 53, in <module>
    setup(
  File "C:\Users\User\AppData\Local\Temp\build-env-lkyfb7qt\lib\site-packages\setuptools\__init__.py", line 153,
in setup
    return distutils.core.setup(**attrs)
  File "C:\Program Files\Python39\lib\distutils\core.py", line 148, in setup
    dist.run_commands()
  File "C:\Program Files\Python39\lib\distutils\dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "C:\Program Files\Python39\lib\distutils\dist.py", line 985, in run_command
    cmd_obj.run()
  File "C:\Users\User\AppData\Local\Temp\build-env-lkyfb7qt\lib\site-packages\wheel\bdist_wheel.py", line 299, in
 run
    self.run_command('build')
  File "C:\Program Files\Python39\lib\distutils\cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "C:\Program Files\Python39\lib\distutils\dist.py", line 985, in run_command
    cmd_obj.run()
  File "C:\Program Files\Python39\lib\distutils\command\build.py", line 135, in run
    self.run_command(cmd_name)
  File "C:\Program Files\Python39\lib\distutils\cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "C:\Program Files\Python39\lib\distutils\dist.py", line 985, in run_command
    cmd_obj.run()
  File "setup.py", line 23, in run
    subprocess.run([os.path.join(project_root, 'mvnw'), 'clean', 'install'], cwd=project_root, check=True)
  File "C:\Program Files\Python39\lib\ntpath.py", line 78, in join
    path = os.fspath(path)
TypeError: expected str, bytes or os.PathLike object, not NoneType

ERROR Backend subproccess exited when trying to invoke build_wheel
```

**Comment by Christopher-Chianelli** [comment]
Probably caused by https://github.com/optapy/optapy/blob/fab133deed98e57a644ddcdd85806cef67bcc845/optapy-core/setup.py#L20 , which is probably undefined in Windows. I'll rewrite it to support Windows build.

**Comment by Christopher-Chianelli** [comment]
Fixed by #16
